### PR TITLE
Don't hit the API to inject sphinx context

### PR DESCRIPTION
**Pull request by stsewd**
This feature flag was used to don't inject private versions,
but we haven't have anyone requesting to enable the feature,
so I'm reusing the flag to skip the API call.

Ref https://github.com/readthedocs/readthedocs.org/issues/6712#issuecomment-592163820
Fixes https://github.com/readthedocs/readthedocs.org/issues/6712

### CODE REVIEW

File: readthedocs/doc_builder/backends/sphinx.py [hunk 1]
```diff
@@ -115,32 +115,22 @@ def get_config_params(self):
...
-            downloads = self.version.get_downloads(pretty=True)
-            subproject_urls = self.project.get_subproject_urls()
-        else:
-            try:
-                versions = self.project.api_versions()
-                if not self.project.has_feature(Feature.ALL_VERSIONS_IN_HTML_CONTEXT):
-                    versions = [
-                        v
-                        for v in versions
-                        if v.privacy_level == PUBLIC
```
Comment (stsewd) [inline readthedocs/doc_builder/backends/sphinx.py:134]: Let me know if you still want to show only public versions if a project has the feature enabled.

**Comment by ericholscher** [comment]
I think this will break some integrations that users have where they are expecting us to inject this data into the context. I don't know the best path forward, but I think we should probably see if we can figure out if people are using this data first.

**Comment by stsewd** [comment]
>  but I think we should probably see if we can figure out if people are using this data first.

The only way to do that is by inspecting the templates used by the user on each build.

**Comment by stsewd** [comment]
We want to address this with something like lazy loading the API calls, since removing this calls isn't a solution I'm closing this PR.

**Pull request closed without merging**